src/main/java/seedu/address/logic/commands/PriorityProjectCommand.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PRIORITY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROJECT;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.Project;
+import seedu.address.model.project.ProjectPriority;
+
+/**
+ * Sets a priority for a project in TaskHub.
+ */
+public class PriorityProjectCommand extends Command {
+    public static final String COMMAND_WORD = "priorityP";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets a priority for a project in TaskHub."
+            + "Parameters: "
+            + PREFIX_PRIORITY + "PRIORITY_LEVEL"
+            + PREFIX_PROJECT + "INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_PRIORITY + "high "
+            + PREFIX_PROJECT + "1";
```

Comment:
I think there is a mistake here. The command format is `priorityP INDEX priority/PRIORITY`, and not `priorityP priority/PRIORITY pr/PROJECT`.